src/ReleaseHistory.md

```diff
@@ -4,6 +4,8 @@
 
 * FEATURE: Add BA3031.EnableClangSafeStack, rename BA3030.UseCheckedFunctionsWithGcc to BA3030.UseGccCheckedFunctions [#663](https://github.com/microsoft/binskim/pull/663)
 * Bump Sarif.Sdk by updating submodule from [fc9a9df to 698adb6](https://github.com/microsoft/sarif-sdk/compare/fc9a9dfb865096b5aaa9fa3651854670940f7459...698adb6365a242c6bb75adde56e3bd4be39c21d7). [#674](https://github.com/microsoft/binskim/pull/674)
+* Introduce first performance rule BA6001.DisableIncrementalLinkingInReleaseBuilds [#667](https://github.com/microsoft/binskim/pull/667)
+* Introduce more performance rules BA6002, BA6004, BA6005, BA6006.
```

Comment:
Please add the PR link to the end of this line: `[#691]([https://github.com/microsoft/binskim/pull/691)`](https://github.com/microsoft/binskim/pull/691)`)